app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/landing.scala.html

```diff
@@ -0,0 +1,100 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import play.api.i18n.Messages
+@import play.api.mvc.Request
+@import play.twirl.api.Html
+
+@import uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers.routes
+@import uk.gov.hmrc.cdsreimbursementclaimfrontend.config.ViewConfig
+
+@this(
+        mainTemplate: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.main_template,
+        pageHeading: uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.page_heading,
+        details : uk.gov.hmrc.cdsreimbursementclaimfrontend.views.html.components.details
+)
+
+@()(implicit messages:Messages, viewConfig: ViewConfig, request: Request[_])
+
+@key = @{"landing"}
+@title = @{messages(s"$key.title")}
+
+@mainTemplate(title = title, userType = None) {
+
+    @pageHeading(title)
+
+    <p class="govuk-body">@messages(s"$key.help.p1")</p>
+    <p class="govuk-body">@messages(s"$key.help.p2")</p>
+
+    <h3>@messages(s"$key.who.heading")</h3>
+    <p class="govuk-body">@messages(s"$key.who.p1")</p>
+    <p class="govuk-body">@Html(messages(s"$key.who.p2", viewConfig.eoriNumberHelpUrl, viewConfig.mrnGuideUrl))</p>
+    <p class="govuk-body">@messages(s"$key.who.p3")</p>
+
+    <ul class="list-bullet">
+     <li>@messages(s"$key.who.bullet1")</li>
```

Comment:
The convention I would like for the service is `l1`, `l2`...etc for bullet point lists.

So these and others would be `$key.who.l1` etc